specs/eventing/control-plane.md

```diff
@@ -0,0 +1,1120 @@
+reference specific core Kubernetes resources; these references may be
+illustrative (i.e. _an implementation on Kubernetes_) or descriptive (i.e. _this
+Kubernetes resource MUST be exposed_). References to these core Kubernetes
+resources will be annotated as either illustrative or descriptive.
+
+This document considers two users of a given Knative Eventing environment, and
+is particularly concerned with the expectations of developers (and language and
+tooling developers, by extension) deploying applications to the environment.
+
+- **Developers** configure Knative resources to implement an event-routing
+  architecture.
+- **Operators** (also known as **platform providers**) provision the underlying
+  event routing resources and manage the software configuration of Knative
+  Eventing and the underlying abstractions.
+
+## RBAC Profile
+
+In order to validate the controls described in
+[Resource Overview](#resource-overview), the following Kubernetes RBAC profile
+may be applied in a Kubernetes cluster. This Kubernetes RBAC is an illustrative
+example of the minimal profile rather than a requirement. This Role should be
+sufficient to develop, deploy, and manage event routing for an application
+within a single namespace. Knative Conformance tests against "MUST", "MUST NOT",
+and "REQUIRED" conditions are expected to pass when using this profile:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: knative-developer
+rules:
+  - apiGroups: ["eventing.knative.dev"]
+    resources: ["broker", "trigger"]
+    verbs: ["get", "list", "create", "update", "patch", "delete"]
+  - apiGroups: ["messaging.knative.dev"]
+    resources: ["channel", "subscription"]
+    verbs: ["get", "list", "create", "update", "patch", "delete"]
+```
+
+In order to support resolving resources which meet the
+[Addressable](./overview.md#addressable) contract, the system controlling the
+[Trigger](#trigger-lifecycle) or [Subscription](#subscription-lifecycle) will
+need _read_ access to these resources. On Kubernetes, this is most easily
+achieved using role aggregation; on systems using Kubernetes RBAC, resources
+which wish to participate in Addressable resolution should provide the following
+`ClusterRole`:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: <component>-addressable-resolver
+  labels:
+    duck.knative.dev/addressable: "true"
+rules:
+  - apiGroups: [<resource apiGroup>]
+    resources: [<resource kind >, <resource kind>/status]
+    verbs: ["get", "list", "watch"]
+```
+
+This configuration advice SHALL NOT indicate a requirement for Kubernetes RBAC
+support.
+
+<!-- TODO: define aggregated API roles
+
+Ref:
+- https://github.com/knative/specs/blob/main/specs/eventing/channel.md#aggregated-channelable-manipulator-clusterrole
+- https://github.com/knative/specs/blob/main/specs/eventing/channel.md#aggregated-addressable-resolver-clusterrole
+-
+-->
+
+## Resource Overview
```

Comment:
Maybe move this up before the RBAC section since it points to this one? I prefer backward references instead of forward references. Plus this reads more like an intro anyway

Reply:
Done.